### PR TITLE
Update normalization docs and add an extra newline for gitbook...

### PR DESCRIPTION
**Pull request by ChristopheDuong**
## What
Fix docs in gitbook
add a warning on normalization limitations (regarding nesting)

### CODE REVIEW

File: docs/architecture/basic-normalization.md [hunk 1]
```diff
@@ -63,6 +63,13 @@ Airbyte uses the types described in the catalog to determine the correct type fo
...
 ### Nesting
 
+{% hint style="warning" %}
+Normalization is still a work in progress and is very basic for the moment. We are still working on improving it.
+Nested Objects and Arrays are not currently handled by Normalization (only flat jsons are properly mapped out as a table).
+You can follow progress as part of this
+[issue](https://github.com/airbytehq/airbyte/issues/886) and its related children.
+{% endhint %}
+
 Basic Normalization attempts to expand any nested arrays or objects it receives into separate tables in order to allow more ergonomic querying of your data.
```
Comment (sherifnada) [inline docs/architecture/basic-normalization.md:73]: should this be removed?

**Review by sherifnada — APPROVED** [review]

**Comment by sherifnada** [comment]
also @ChristopheDuong make sure to update the PR name when merging to master to reflect that this is really an update to normalization docs :)

File: docs/architecture/basic-normalization.md (hunk 1 above)
Comment (ChristopheDuong) [reply docs/architecture/basic-normalization.md:73]: That's what normalization should do once we fix FE/BE issues with the catalog so we can keep it for the meantime?

Comment (sherifnada) [reply docs/architecture/basic-normalization.md:73]: it's just that it conflicts with the text you just added about arrays not yet being supported, which could confuse a user. I'd say we should remove it or edit it then re-add it in the PR to support nested

**Pull request merged**